src/components/NcActions/NcActions.vue

```diff
@@ -1407,6 +1445,22 @@ export default {
 			}
 		},
 
+		pushSubmenu(submenu) {
```

Comment:
Need a check here if slot provided. Otherwise that would be a breaking, if a developer accidentally (🙈) passed a String instead of boolean, like:
```vue
<NcActionButton :is-menu="hasSubmenu && activeMenu">Option 2</NcActionButton>
```
```js
const hasSubmenu = true
const activeMenu = 'option2'
```

![2024-11-12_11h37_17](https://github.com/user-attachments/assets/fbe074e1-2bb7-4b5d-aa7b-61b2765d04a4)

Reply:
Maybe add `has-menu` instead of reusing `is-menu` ?